src/components/_variables.scss

```diff
@@ -0,0 +1,9 @@
+$font: "DINOT", sans-serif;
+
+$white: #fff;
+$grey: #ccc;
```

Comment:
I think these colors are not the ones used by designers. Double check, we normally use bg with opacity, instead of direct hex color

Reply:
I'll check, I explicitly used hex over rgba because it allowed me to make use of the same halo technique used in m-dash-ui.  If we don't need the halo, I'll swap back to rgba 👍